src/main/java/com/expediagroup/hiveberg/IcebergFilterFactory.java

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright (C) 2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.hiveberg;
+
+import static org.apache.iceberg.expressions.Expressions.and;
```

Comment:
I'd recommend changing your import order defaults to match Iceberg's for this project. It will make it easier to move code back and forth. From what I can tell they just have them in alphabetical order, no grouping other than non-static first and then static (with a line break between).